app/models/concerns/course/assessment/submission/notification_concern.rb

```diff
@@ -23,8 +23,11 @@ def send_submit_notification
     # as the student as it's the only way to indicate that the submission belongs to the student.
     # In such case, there is no need to send a notification to the course staff that there is
     # a new submission to be graded since it was submitted by the course staff anyway.
+    is_course_koditsu_enabled = assessment.course.component_enabled?(Course::KoditsuPlatformComponent)
+    is_assessment_koditsu_enabled = assessment.is_koditsu_enabled? && assessment.koditsu_assessment_id
+
     return unless creator == updater
-    return if assessment.autograded?
+    return if assessment.autograded? || (is_course_koditsu_enabled && is_assessment_koditsu_enabled)
```

Comment:
Why don't notify?

Reply:
this submission fetching serves only the teacher; students do not need to know if their submissions have been fetched properly from Koditsu to Coursemology

what they need to be concerned about is only that they finished their exam in Koditsu and wait for the results later (a.k.a. being graded)

Reply:
What are you talking about? This notification is not to the student, the submission notification is to the instructor.

Reply:
oh sorry I terribly misunderstood what this is doing..
I really thought this notification is for student.. (since there's this thing called returning if assessment.autograded?). Have reverted this back